BirthdayGreetingsTest/api.js

```diff
@@ -64,22 +59,22 @@ app.put("/birthdays/add-friend", (req, res) => {
   const { lastName, firstName, dateOfBirth, email } = req.body;
   addNewBirthdayFriendToDB(lastName, firstName, dateOfBirth, email)
     .then((result) => {
-      res.status(200).send(result);
+      res.status(201).send(result);
     })
     .catch((err) => {
-      res.status(500).send(err);
+      res.status(err.status).send(err);
     });
 });
 
-app.delete("/birthdays/delete-friend/:email", (req, res) => {
+app.delete("/birthdays/delete-friend", (req, res) => {
```

Comment:
Same with this one, can you come up with something that better meets the REST standard?